src/background.ts

```diff
@@ -87,16 +87,25 @@ async function sendTask(tabId: number, messageId: string, data: string): Promise
     return;
   }
 
-  try {
```

Comment:
I see you unwrapped `await getHosts()` from `try catch` block. Not sure why. Maybe rewrap it to maintain consistency with previous code?

Reply:
Oops, nice point, I think I got lost during indentation fixes here.